test/test_helper.rb

```diff
@@ -7,6 +7,7 @@
 ActiveRecord::Migrator.migrations_paths = [File.expand_path("../test/dummy/db/migrate", __dir__)]
 require "rails/test_help"
 require "rails/test_unit/reporter"
+require "pry"
```

Comment:
**thought:** do we really need `pry` here for debugging? Would `binding.irb` be enough? 

In pbbuilder, for example, irb doesn't work. But for this gem we can easily debug without `pry`.

Reply:
Sure yanked it out.